manifests/pub_dir.pp

```diff
@@ -15,7 +15,7 @@
     priority            => '05',
     docroot             => '/var/www/html',
     options             => ['SymLinksIfOwnerMatch'],
-    additional_includes => ['/etc/pulp/vhosts80/*.conf'],
+    additional_includes => ['/etc/httpd/conf.d/vhosts80/*.conf'],
```

Comment:
`puppet-pulp` uses `${apache::confd_dir}`, but here we hardcode `/etc/httpd/conf.d`? (Debian ships httpd config in `/etc/apache2` ;))

Reply:
this comment remains